build/DependencyVersions.props

```diff
@@ -12,12 +12,12 @@
     <!-- We'll usually want to keep these versions in sync, but we may want to diverge in some
          cases, so use separate properties but derive one from the other unless we want to
          explicitly use different versions. -->
-    <CLI_NETSDK_Version>2.0.1-servicing-20170912-1</CLI_NETSDK_Version>
+    <CLI_NETSDK_Version>2.0.2-vspre-20170921-5</CLI_NETSDK_Version>
```

Comment:
servicing?

Reply:
The vspre is actually newer for now. The problem is that we haven't had builds out in a while, because of signing. I will update once builds start succeeding.